lib/src/main/java/graphql/nadel/engine/blueprint/NadelExecutionBlueprintFactory.kt

```diff
@@ -471,6 +471,21 @@ private class Factory(
         )
     }
 
+    private fun makePartitionInstruction(
+        parentType: GraphQLObjectType,
+        field: GraphQLFieldDefinition,
+    ): List<NadelPartitionInstruction> {
+        val partitionDefinition = NadelDirectives.createPartitionDefinition(field)
+            ?: return emptyList()
+
+        return listOf(
+            NadelPartitionInstruction(
+                location = makeFieldCoordinates(parentType, field),
+                pathToPartitionArg = partitionDefinition.pathToSplitPoint,
+            )
+        )
```

Comment:
Shouldn't this store more info like whether the output type is Payload like? etc.

Though maybe leave the changes here minimal because I'm exploring merging validation & blueprint code together in another branch.

Reply:
True. All this static information should live in the Blueprint.
I'll wait for your refactor to make the change.